app/decorators/sponsorship_decorator.rb

```diff
@@ -1,11 +0,0 @@
-class SponsorshipDecorator < ApplicationDecorator
-  def level_background_color
```

Comment:
I checked for `level_background_color` in the code, but it seems like this decorator was not used anywhere, however we were testing decorators in general by using the SponsorshipDecrator. I remove these tests later on.